translationtable/zfinslim.yaml

```diff
@@ -0,0 +1,3 @@
+---
+# zfinslim.yaml
+"": ""  # example
```

Comment:
Purpose?

Reply:
This was autogenerated by dipper-etl.py. Not sure why it made this file, but it probably needs to be in the repo